app/src/main/java/zegreanu/cristi/criminalintent/CrimeLab.kt

```diff
@@ -0,0 +1,33 @@
+package zegreanu.cristi.criminalintent
+
+import android.content.Context
+import java.util.*
+
+// Context is for chapter 14
+class CrimeLab private constructor(context: Context) {
```

Comment:
Although the implementation in the chapter requires all these steps, I'd say we can leverage kotlin's object keyword:
https://kotlinlang.org/docs/reference/object-declarations.html#object-declarations